src/elements/Label/Label.js

```diff
@@ -35,31 +34,32 @@ function Label(props) {
     useKeyOnly(circular, 'circular'),
     useKeyOnly(floating, 'floating'),
     useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(image, 'image'),
     useKeyOnly(empty, 'empty'),
     useKeyOnly(tag, 'tag'),
     useValueAndKey(attached, 'attached'),
     useKeyOrValueAndKey(corner, 'corner'),
     useKeyOrValueAndKey(pointing, 'pointing'),
+    /* Don't change order of `above` and `below`, they must be after `pointing` */
+    useKeyOnly(above, 'above'),
+    useKeyOnly(below, 'below'),
     useKeyOrValueAndKey(ribbon, 'ribbon'),
-    // TODO how to handle image child with no image class? there are two image style labels.
-    (image || childrenUtils.someByType(children, Image) || childrenUtils.someByType(children, 'img')) && 'image',
```

Comment:
I think we keep image shorthand and make the assumption that if you use it, you want the `image` class added.  This way, shorthand produces this type:

![image](https://cloud.githubusercontent.com/assets/5067638/18762362/49471066-80bd-11e6-8de1-e579e1ffcd25.png)


The user can define the markup without the image prop if they want this format:

![image](https://cloud.githubusercontent.com/assets/5067638/18762376/5c643430-80bd-11e6-8f9e-a022e60d490c.png)

### Compare to the Header

This is consistent with the assumptions we had to make in other components, like the icon Header.  In this case, we opted for the shorthand to _not_ generate the icon class (icon type Header):

![image](https://cloud.githubusercontent.com/assets/5067638/18762422/8d4ff476-80bd-11e6-9940-a1cdce5b4f98.png)

![image](https://cloud.githubusercontent.com/assets/5067638/18762449/a7951230-80bd-11e6-95ed-7e9ccc808fe0.png)